data/sql/updates/pending_db_world/rev_1599848604760949500.sql

```diff
@@ -0,0 +1,12 @@
+DELETE FROM `creature_formations` WHERE `leaderGUID`=9203;
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES 
+(9203, 9203, 0, 0, 2, 0, 0),
+(9203, 9204, 3, 45, 2, 0, 0),
+(9203, 9205, 3, 315, 2, 0, 0);
+
+DELETE FROM `creature_addon` WHERE guid IN (9204,9205);
+DELETE FROM `waypoint_data` WHERE id IN (92040,92050);
+UPDATE `creature` SET `MovementType`=0, `SpawnDist`=0 WHERE `id` IN (2477,7170);
```

Comment:
There is no such thing as SpawnDist :D 
Update your core and change this to wander_distance :P